package.json

```diff
@@ -114,32 +97,27 @@
     "typed-emitter": "^2.1.0",
     "typescript": "^4.9.4",
     "ud": "^3.3.1",
-    "ud-kefir": "^2.0.1",
-    "vinyl-source-stream": "^2.0.0",
-    "watchify": "^4.0.0"
+    "ud-kefir": "^2.0.1"
   },
   "description": "Library for browser extensions to interact with Gmail.",
   "license": "(MIT OR Apache-2.0)",
   "main": "src/inboxsdk-js/inboxsdk.js",
   "name": "inboxsdk",
   "private": true,
-  "redirectify": {
-    "base": ".",
-    "dir": "overrides"
-  },
   "scripts": {
     "transpile-flow": "babel -d packages/core/src --source-type module --ignore '*.test.js' --no-copy-ignored src",
+    "typecheck": "yarn run flow_check && tsc",
     "typedefs": "yarn transpile-flow && tsc -p tsconfig.declarations.json",
     "prepare": "husky install",
     "publish-npm": "gulp clean && gulp --production --minify && yarn workspace @inboxsdk/core npm publish && git push --follow-tags",
     "flow_check": "flow check",
     "lint": "eslint . --report-unused-disable-directives",
-    "lint-fix": "eslint . --ext js,jsx,ts,tsx,flow --fix",
+    "lint:fix": "eslint . --ext js,jsx,ts,tsx,flow --fix",
     "prettier-check": "prettier --check .",
     "prettier-fix": "prettier --write .",
     "puppeteer": "jest -c test/chrome/jest.config.js --runInBand",
     "start": "gulp default --watch --reloader",
-    "test": "yarn run lint && yarn run prettier-check && yarn run flow_check && tsc"
+    "test": "jest"
```

Comment:
prettier-check step is missing now. Maybe make lint call it, or otherwise make sure ci calls it.

Reply:
[0cc6f3f](https://github.com/InboxSDK/InboxSDK/pull/881/commits/0cc6f3fff30d86ecdc6dc79ae7df1b3cc82df7a3)